### PR TITLE
changes for getting started launch doc pages

### PR DESCRIPTION
**Pull request by mirandaauhl**
# Description

adding a docker section to getting started launch methods

# Version

Which documentation version does this PR apply to?

- [x ] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

**Comment by mirandaauhl** [comment]
Alright friends, here is the first set of updates.

File: timescaledb/getting-started/launch-timescaledb/index.md [hunk 1]
```diff
@@ -1,75 +1,23 @@
...
-</highlight>
+Creating a TimescaleDB Docker instance can be a quick and useful alternative to using Timescale 
+Cloud but requires maintaining the database. Additionally, you will need to have Docker installed on 
+your local machine to use this option. 
 
-### Step 1: Create a Timescale Cloud account
 
-Sign up for Timescale Cloud by visiting [console.cloud.timescale.com][cloud-signup].
+Methods:
+* [Launch TimescaleDB using the Cloud][launch-timescaledb-cloud]
```
Comment (zseta) [inline timescaledb/getting-started/launch-timescaledb/index.md:19]:
```suggestion
* [Launch TimescaleDB using Timescale Cloud][launch-timescaledb-cloud]
```

File: timescaledb/getting-started/launch-timescaledb/launch-timescaledb-cloud.md [hunk 2]
```diff
@@ -1,49 +1,38 @@
...
-email. If you don't receive this link, first check your spam folder
-and, failing that, [contact us][contact-timescale].
+You need to confirm your account by clicking on the link you receive via email. If you 
+don't receive this link, first check your spam folder and, failing that, [contact us][contact-timescale].
 
 ### Step 2: Create your first service
 
-After confirming your account, visit the
-[Timescale Cloud console][cloud-console] and log in with your credentials.
+After confirming your account, there should be a link that will redirect you to your
```
Comment (zseta) [inline timescaledb/getting-started/launch-timescaledb/launch-timescaledb-cloud.md:16]: in the email I guess?

File: timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md [hunk 3]
```diff
@@ -1,75 +1,34 @@
...
-For this Getting Started tutorial, we'll use Timescale Cloud to get you up
-and running quickly. Timescale Cloud combines the power and reliability
-of TimescaleDB with a fully managed, cloud-native experience that is easy to
-start and less expensive to operate.
+### Step 1: Download and install Docker
 
-Creating a Timescale Cloud account is fast and simple, allowing you to explore
-TimescaleDB free for 30 days, no credit card required. We're just excited
-that you're ready to dive into the world of time-series data with TimescaleDB.
+To install and launch TimescaleDB using Docker, you will first need to install Docker on your machine.
```
Comment (zseta) [inline timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md:5]:
```suggestion
To install and launch TimescaleDB using Docker, you first need to install Docker on your machine. 
```

File: timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md [hunk 4]
```diff
@@ -1,75 +1,34 @@
...
-and running quickly. Timescale Cloud combines the power and reliability
-of TimescaleDB with a fully managed, cloud-native experience that is easy to
-start and less expensive to operate.
+### Step 1: Download and install Docker
 
-Creating a Timescale Cloud account is fast and simple, allowing you to explore
-TimescaleDB free for 30 days, no credit card required. We're just excited
-that you're ready to dive into the world of time-series data with TimescaleDB.
+To install and launch TimescaleDB using Docker, you will first need to install Docker on your machine. 
+You will find information on downloading and installing Docker on their [getting started page][docker-install].
```
Comment (zseta) [inline timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md:6]:
```suggestion
You find information on downloading and installing Docker on their [getting started page][docker-install]. 
```

File: timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md [hunk 5]
```diff
@@ -1,75 +1,34 @@
...
+Once you have Docker installed, you can create a TimescaleDB container instance. 
 
-### Step 1: Create a Timescale Cloud account
+### Step 2: Create TimescaleDB instance within a Docker container
 
-Sign up for Timescale Cloud by visiting [console.cloud.timescale.com][cloud-signup].
+This tutorial will show you how to create a docker container instance using the [official TimescaleDB docker image][timescale-docker]. While you can use `docker pull timescale/timescaledb` to add the TimescaleDB image to your Docker library, you do not have to. Docker will automatically pull the image if it is not already in your Docker library when running the `docker run` command below. 
 
-Provide your full name, email address, and a strong password to start:
+To create a docker container using this image, you need to run the following code. Once you run this command, you will have a TimescaleDB instance on Docker ready to connect to.
```
Comment (zseta) [inline timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md:14]:
```suggestion
To create a Docker container using this image, you need to run the following code. Once you run this command, you will have a TimescaleDB instance on Docker ready to connect to. 
```

File: timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md [hunk 6]
```diff
@@ -1,75 +1,34 @@
...
+- The `--name timescaledb` is a flag that allows you to specify the container’s name, which is “timescaledb” in this case.
+- The `-p 127.0.0.1:5432:5432` lets Docker know you want to connect your local machines (host 127.0.0.1) 5432 port to the containers 5432 port (the default PostgreSQL port). By doing this, you can connect to the PostgreSQL 5432 port within the container from our local machine's 5432 port.  
+- The `-e POSTGRES_PASSWORD=password` is a flag that sets the PostgreSQL database's password
+-  And finally, the `timescale/timescaledb:lastest-pg14` lets Docker know the image it should use when creating the container
 
-After confirming your account, visit the
-[Timescale Cloud console][cloud-console] and log in with your credentials.
+The last portion, `latest-pg14`, of the command `timescale/timescaledb:lastest-pg14` specifies the TimescaleDB version should be the latest, and the PostgreSQL version should be 14. You can select different versions of TimescaleDB and PostgreSQL. However, we recommend using the latest version of TimescaleDB and PostgreSQL that we support.
```
Comment (zseta) [inline timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md:30]:
```suggestion
The last portion, `latest-pg14`, of the command `timescale/timescaledb:lastest-pg14` specifies that the TimescaleDB version should be the latest, and the PostgreSQL version should be 14. You can select different versions of TimescaleDB and PostgreSQL. However, we recommend using the latest version of TimescaleDB and PostgreSQL that we support. 
```

**Review by zseta — COMMENTED** [review]
Great work @mirandaauhl ! Just added minor fixes

File: timescaledb/getting-started/launch-timescaledb/launch-timescaledb-cloud.md (hunk 2 above)
Comment (mirandaauhl) [reply timescaledb/getting-started/launch-timescaledb/launch-timescaledb-cloud.md:16]: I mention the email in line 11 12ish, I figure that would be enough?

File: timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md [hunk 7]
```diff
@@ -1,75 +1,34 @@
...
 
-<highlight type="tip">
-If you would prefer to set up TimescaleDB in your own environment and follow
-along with the rest of the _Getting Started_ tutorial, please see our documentation
-for [installing self-hosted TimescaleDB](/install/latest/self-hosted/).
-</highlight>
+Once you have Docker installed, you can create a TimescaleDB container instance. 
 
-### Step 1: Create a Timescale Cloud account
+### Step 2: Create TimescaleDB instance within a Docker container
```
Comment (mirandaauhl) [inline timescaledb/getting-started/launch-timescaledb/launch-timescaledb-docker.md:10]:
```suggestion
### Create TimescaleDB instance within a Docker container
```

**Comment by charislam** [comment]
@mirandaauhl looks great! I had some suggestions for aligning this to the style and voice we've defined for docs. Tried to do it as GitHub suggestions but it turned into a giant mess so I made a separate PR to merge into this branch, which you can review: #865

**Comment by mirandaauhl** [comment]
Okay team, I think we are done with this section, please let me know if you have any other comments!

**Comment by mirandaauhl** [comment]
actually just going to go ahead and push to the main getting started branch :)

**Pull request merged**